languages/elixir/concepts/list-comprehensions/about.md

```diff
@@ -30,7 +30,7 @@ end
 # => [{0, 0}, {0, 1}, {1, 0}, {1, 1}]
 ```
 
-The value in the do-block is inserted into the _collectable_ **for each value generated** from the _enumerable_. It may or may not include variables bound inside of the comprehension.
```

Comment:
I found this sentence very surprising and didn't understand it at first, it seems to not fix the context. Then I realized that this is most likely about the general property of do-blocks and anonymous functions. Maybe it doesn't need to be said here at all?

Reply:
Ya, When looking at this one sentence, it now seems out of place